utils/update-checkout

```diff
@@ -52,6 +53,20 @@ def update_working_copy(repo_path):
         else:
             check_call([ "svn", "update" ])
 
+def obtain_additional_swift_sources():
+    additional_repos = {
+        'llvm': 'https://github.com/apple/swift-llvm.git',
+        'clang': 'https://github.com/apple/swift-clang.git',
+        'lldb': 'https://github.com/apple/swift-lldb.git',
```

Comment:
Should all repos be cloned or only the minimal subset needed for building Swift (`swift` obviously, `swift-llvm` and `swift-clang`) ?